zetaclient/testutils/testdata.go

```diff
@@ -55,27 +42,6 @@ func ComplianceConfigTest() config.ComplianceConfig {
 	}
 }
 
-// SaveTrimedEVMBlockTrimTxInput trims tx input data from a block and saves it to a file
-func SaveEVMBlockTrimTxInput(block *ethrpc.Block, filename string) error {
```

Comment:
Same. It's been used to save trimmed block.

Reply:
6e57c8f9